### PR TITLE
ascii art to SVG in HTML mode

### PR DESCRIPTION
**Pull request by thomas-fossati**
Needs [aasvg](https://github.com/martinthomson/aasvg#usage)

### CODE REVIEW

**Review by OR13 — APPROVED** [review]

File: draft-birkholz-scitt-architecture.md [hunk 1]
```diff
@@ -215,30 +215,30 @@ Reputable Issuers are thus incentivized to carefully review their Statements bef
...
+Issuer       --> Statement    Envelope    | DID Key Manifest |
+                   \           /          | (decentralized)  |
+                    \         /           '--+---------+-----'
+                     \_______/               |         |
+                         |                   |         |
+                         v        signature  |         |
+                       Claim  <-------------'          |
+                         |                             |
+                         |   Receipt   .--------.      |
+Transparency -->         +-------------+ Ledger +-.    |
```
Comment (OR13) [inline draft-birkholz-scitt-architecture.md:231]: "registry" ?

**Comment by henkbirkholz** [comment]
@thomas-fossati: is this PR superseded by #23? Could you check?

**Comment by thomas-fossati** [comment]
> @thomas-fossati: is this PR superseded by #23? Could you check?

Looks like, I see some awesome stuff there!

**Comment by thomas-fossati** [comment]
Superseded by #23

**Pull request closed without merging**